examples/vm-from-user-image-managed-disk/main.tf

```diff
@@ -0,0 +1,116 @@
+# Configure the Microsoft Azure Provider
+# ARM_SUBSCRIPTION_ID, ARM_CLIENT_ID, ARM_CLIENT_SECRET and ARM_TENANT_ID environment variables assumed to be set 
+# If environment variables not set, declare in azurerm provider
+provider "azurerm" {}
+
+variable "publicIPAddressType" {
+  default = "Dynamic"
+}
+
+variable "osTypeMap" {
+  description = "This is the OS that your VM will be running (value constraints)"
+  type        = "map"
+
+  default = {
+    Windows = "windows"
+    Linux   = "linux"
+  }
+}
+
+variable "newOrExistingVnetMap" {
+  description = "Select if this template needs a new VNet or will reference an existing VNet (value constraints)"
+  type        = "map"
+
+  default = {
+    new      = "1"
+    existing = "0"
+  }
```

Comment:
given this is an example, why don't we remove this, and default to creating a new network instead?